src/compiled/glift_combined.js

```diff
@@ -10565,6 +10578,59 @@ glift.controllers.BoardEditorMethods = {
 /**
  * A GameViewer encapsulates the idea of traversing a read-only SGF.
  */
+glift.controllers.gameSlice = function(sgfOptions) {
+  var ctrl = glift.controllers;
+  var baseController = glift.util.beget(ctrl.base());
+  var newController = glift.util.setMethods(baseController,
+      ctrl.GameSliceMethods);
+  newController.drawTo = sgfOptions.drawTo || [];
+  newController.initOptions(sgfOptions);
+  return newController;
+};
+
+glift.controllers.GameSliceMethods = {
+
+  initialize: function(treepath) {
+    var rules = glift.rules;
+    var initTreepath = treepath || this.initialPosition;
+    this.treepath = rules.treepath.parsePath(initTreepath);
+
+    var initialPosition = this.treepath.length; // used later
+    var nextTreepath = this.drawTo - this.treepath.length;
+    if (this.nextMovesPath.length > 0) {
```

Comment:
Just some self review: I shouldn't overload `nextMovesPath`, I'd like to allow `drawTo` to contain a position or a variation.